target/linux/ramips/image/mt7621.mk

```diff
@@ -203,6 +203,36 @@ define Device/rb750gr3
 endef
 TARGET_DEVICES += rb750gr3
 
+define Device/rbm33g
+  DTS := RBM33G
+  IMAGE_SIZE := $(ralink_default_fw_size_16M)
```

Comment:
your firmware partition is 16515072 bytes but ralink_default_fw_size_16M is 16121856. Please use `IMAGE_SIZE := 16128k` here.

Reply:
Yep, of course. I should have adjusted that before pushing